### DISCUSSION TITLE
每日副本无法找到我想要的那个副本

### DISCUSSION THREAD

**Discussion by Alice782k** [LmeSzinc/StarRailCopilot#409] [Q&A]
由于版本更新，有新的行迹材料出现，导致SRC翻页的时候只翻到中间（我原来想打的那个副本的位置），无法继续往下翻页了，请问有什么解决方法吗？是否可以让SRC将所有副本选项都浏览一遍之后再选择呢？

**Comment by lenze374** [comment]
该更新了！
